platforms/chibios/_pin_defs.h

```diff
@@ -292,3 +292,63 @@
 #    define K14 PAL_LINE(GPIOK, 14)
 #    define K15 PAL_LINE(GPIOK, 15)
 #endif
+
+#if defined(MCU_LPC)
+//   GPIO Port 0
+#    define P0_0 PAL_LINE(IOPORT0, 0)
+#    define P0_1 PAL_LINE(IOPORT0, 1)
+#    define P0_2 PAL_LINE(IOPORT0, 2)
+#    define P0_3 PAL_LINE(IOPORT0, 3)
+#    define P0_4 PAL_LINE(IOPORT0, 4)
+#    define P0_5 PAL_LINE(IOPORT0, 5)
+#    define P0_6 PAL_LINE(IOPORT0, 6)
+#    define P0_7 PAL_LINE(IOPORT0, 7)
+#    define P0_8 PAL_LINE(IOPORT0, 8)
+#    define P0_9 PAL_LINE(IOPORT0, 9)
+#    define P0_10 PAL_LINE(IOPORT0, 10)
+#    define P0_11 PAL_LINE(IOPORT0, 11)
+#    define P0_12 PAL_LINE(IOPORT0, 12)
+#    define P0_13 PAL_LINE(IOPORT0, 13)
+#    define P0_14 PAL_LINE(IOPORT0, 14)
+#    define P0_15 PAL_LINE(IOPORT0, 15)
+#    define P0_16 PAL_LINE(IOPORT0, 16)
+#    define P0_17 PAL_LINE(IOPORT0, 17)
+#    define P0_18 PAL_LINE(IOPORT0, 18)
+#    define P0_19 PAL_LINE(IOPORT0, 19)
+#    define P0_20 PAL_LINE(IOPORT0, 20)
+#    define P0_21 PAL_LINE(IOPORT0, 21)
+#    define P0_22 PAL_LINE(IOPORT0, 22)
+#    define P0_23 PAL_LINE(IOPORT0, 23)
+//   GPIO Port 1
```

Comment:
Does the whole LPC family have only pins 0…23 on P0, or is that specific to a particular chip in that family?

Reply:
The other chips have the same amount or less in that family. It depends on the size of the chip.